backend/vkt/src/main/java/fi/oph/vkt/service/PublicEnrollmentService.java

```diff
@@ -117,6 +117,7 @@ private PublicEnrollmentDTO createEnrollmentDTO(final Enrollment enrollment) {
       .postalCode(enrollment.getPostalCode())
       .town(enrollment.getTown())
       .country(enrollment.getCountry())
+      .hasPaymentLink(enrollment.getPaymentLinkHash() != null)
```

Comment:
Teoriassa kai mahdollista että luodaan jonosta nostetulle maksulinkki, virkailija peruu ilmoittautumisen (miksi?) ja henkilö ilmoittautuu uudelleen, jolloin stepperiä ei taitais ilmoittautuessa näkyä.

Pitäiskö tässä tsekata sen sijaan juuri tuo status, mikä tarkistus oli frontin päässä aiemmin? Ja nimeäis tän ehkä `isShiftedFromQueue` ?

Reply:
Hmm, tarvittais uusi status tyyliin `PAID_SHIFTED_FROM_QUEUE`. Nyt kun se on pelkkä `PAID` niin onko mitään keinoa erottaa maksulinkin kautta maksettua 🤔